addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -163,18 +163,18 @@
                     <ul t-if="aside_slide.slide_resource_ids or aside_slide.question_ids" class="list-group px-2 mb-1 list-unstyled">
                         <t t-foreach="aside_slide.slide_resource_ids" t-as="resource">
                             <li class="pl-4">
-                                <a t-if="can_access and resource.link" t-att-href="resource.link" target="new" class="text-decoration-none small">
+                                <a t-if="can_access and resource.resource_type == 'url'" t-att-href="resource.link" target="new" class="text-decoration-none small">
                                     <i class="fa fa-link mr-1"/><span t-field="resource.name"/>
                                 </a>
-                                <span t-else="" class="text-decoration-none text-muted small">
+                                <span t-elif="resource.resource_type == 'url'" class="text-decoration-none text-muted small">
                                     <i class="fa fa-link mr-1"/><span t-field="resource.name"/>
                                 </span>
                             </li>
                         </t>
                         <div class="o_wslides_js_course_join pl-4" t-if="aside_slide.slide_resource_ids">
                             <t t-if="is_member or aside_slide.channel_id.can_publish">
                                 <li t-foreach="aside_slide.slide_resource_ids" t-as="resource">
```

Comment:
same remark as above, we will have empty `<li>` so similar thing should be done `file` resource type